Oqtane.Server/Startup.cs

```diff
@@ -136,10 +140,10 @@ public void ConfigureServices(IServiceCollection services)
                 ));
             services.AddDbContext<TenantDBContext>(options => { });
 
-            services.AddIdentityCore<IdentityUser>(options => { })
-                .AddEntityFrameworkStores<TenantDBContext>()
-                .AddSignInManager()
-                .AddDefaultTokenProviders();
+            if (FeatureManager.IsEnabled("Identity"))
+            {
+                services.AddSingleton<IServerStartup, IdentityStartup>();
```

Comment:
Is this should be Singleton?

Reply:
Nah seems I did it wrong, the `IServerStartup` will register it's services automatically, but how we can controller the registration in `Startup`?